WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/PointOfSalePaymentSuccessView.swift

```diff
@@ -18,57 +18,71 @@ struct PointOfSalePaymentSuccessView: View {
                         insertion: .move(edge: .trailing).combined(with: .opacity),
                         removal: .move(edge: .trailing).combined(with: .opacity)))
             } else {
-                ZStack {
-                    VStack(alignment: .center, spacing: Constants.headerSpacing) {
-                        successIcon
-                            .renderedIf(!dynamicTypeSize.isAccessibilitySize)
-                            .scaleEffect(isViewLoaded ? 1 : 0)
-                            .opacity(isViewLoaded ? 1 : 0)
+                HStack(alignment: .center) {
+                    Spacer()
+                    successView
+                    Spacer()
+                }
+                .padding([.leading, .trailing], dynamicTypeSize.isAccessibilitySize ? nil : 8)
```

Comment:
🗒️ Note for reviewer: this is to recover the padding from the removal of the following in the `TotalsView` for the success view (which the receipt needs):

```
.padding([.leading, .trailing],
         dynamicTypeSize.isAccessibilitySize ? nil :
            cardReaderViewLayout.sidePadding)
.padding(.bottom,
         dynamicTypeSize.isAccessibilitySize ? nil :
            cardReaderViewLayout.bottomPadding)
.padding(.top, dynamicTypeSize.isAccessibilitySize ? nil : cardReaderViewLayout.topPadding)
```

Reply:
The diffs in GitHub aren't shown cleanly, the original ZStack for the success view is now in a private var `successView` so that the HStack + Spacer layout is more obvious from `TotalsView`.